ahoopes16/helianthus
# Provide Information for All Content Pages

# Summary
As I was creating a separate branch for each page, I was going through them quickly enough that I realized I would much rather do one big pull request to add *all* of the content that was remaining. It felt simpler. :)

If accepted, this PR will:
- Fill out the placeholder for the RSVP page
- Fill out the information for the FAQ page
- Fill out the information for the Travel page
- Fill out the information for the Lodging page
- Fill out the information for the Engagement Photos page
- Fill out the information for the Points of Interest page
- Fill out the information for the Gifts
- Provide a link to the About page when someone clicks on the logo in the header
- Extract some common components used between most if not all of the different content pages
- Close #13
- Close #14 
- Close #15 
- Close #17 
- Close #18 
- Close #19 
- Close #31 

# How to Test
1. Pull down this branch and `yarn && yarn start`
2. Visit http://localhoist:3000
3. Check out each page! Make sure that there is some kind of content on all of them and you can get to each page
4. Right now, this is only for the desktop version, so it doesn't look nice when you shrink the viewport. That's okay for now. :)
